src/pages/home/index.jsx

```diff
@@ -1,15 +1,43 @@
-import React from 'react'
-
-import NotificationBar from 'components/notification-bar'
-import HeaderContainer from 'components/common/header-container'
+import React from "react";
+import styled from "styled-components";
+import NotificationBar from "components/notification-bar";
+import HeaderContainer from "components/common/header-container";
+import Card02 from "components/common/cards/card02";
+import Card03 from "components/common/cards/card03";
+import Cards from "components/common/cards/card01";
 
```

Comment:
take styling out of here and keep it clean. To follow standard protocols of react you need to keep in mind that each component is responsible for only one thing. make a section component and put it here like mine. no styling here for now.